### ISSUE TITLE
Mouse and keyboard events are not being captured.

### ISSUE THREAD

**Issue by dipeshji** [kwhat/jnativehook#400]
On mac M1 architecture mouse and keyboard events are not being captured.

**Comment by kwhat** [comment]
You are going to have to provide more information than it does not work.  From what I've been told, 2.2.1 works fine on the M1.  Please provide debug level log output illustrating the problem.  I suspect you have not enabled access for accessibility devices.

**Comment by dipeshji** [comment]
Can you please let me know what access permissions are required in M1?

**Comment by dipeshji** [comment]
Are you talking about this dependency? 
`<dependency>
			<groupId>com.github.kwhat</groupId>
			<artifactId>jnativehook</artifactId>
			<version>2.2.1</version>
		</dependency>`

**Comment by kwhat** [comment]
> Are you talking about this dependency?

Yes, that should work fine with an M1 mac but I have no way of testing it.  Check your console logs for errors, something else maybe wrong.

**Comment by dipeshji** [comment]
I'm not getting any error, but also it's not capturing the events. I think their might be some issues with permissions. So can you please let me know the required permissions?

**Comment by dipeshji** [comment]
Attached screenshots of accessibility permissions. 
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/44906956/157823811-259ef75f-8da6-4aaa-b93e-bccf768ff8ae.png)
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/44906956/157823815-374c0ff1-0b89-4150-abed-487128913205.png)
.

**Comment by kwhat** [comment]
You called `GlobalScreen.registerNativeHook()` right?  Did you test running the jar from the terminal?

**Comment by dipeshpatidar** [comment]
While running the jar, i got the error `Error: Unable to initialize main class com.fourtysevenbillion.timesheet.Timesheet
Caused by: java.lang.NoClassDefFoundError: com/github/kwhat/jnativehook/NativeHookException`
